101-automation-with-modules-and-runbook-execution/azuredeploy.json

```diff
@@ -34,44 +34,46 @@
     "Password": {
       "type": "securestring",
       "metadata": {
-        "description": "aduser password"
+        "description": "Azure AD Password to authenticate into automation runbook"
       }
     },
-	 "WebAppName": {
+    "jobId": {
       "type": "string",
       "metadata": {
-        "description": "This is the name of the web app that may be provided dynamically after creating."
+        "description": "The GUID for the runbook job to be started."
       }
     },
-	 "WebAppResourceGroup": {
+    "SubscriptionId": {
       "type": "string",
       "metadata": {
-        "description": "This is the name of the resource group in which web app resides."
+        "description": "It is the Id of the subscription in which webapp resides"
       }
     },
-    "jobId": {
-      "type": "string",
-      "metadata": {
-          "description": "The GUID for the runbook job to be started."
-      }
-  },
-  "SubscriptionName":{
-    "type": "string",
-  "metadata": {
-      "description": "SubscriptionName"
-  }
-  }
+    "_artifactsLocation": {
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts - your defaultValues need to be fixed